### ISSUE TITLE
[NEW FEATURE] restart should use a different colour

### ISSUE THREAD

**Issue by mrjackwills** [mrjackwills/oxker#4]
**Is your feature request related to a problem? Please describe.**
When restarting a container, the columns are still coloured as green

**Describe the solution you'd like**
A container in a restarting state should use a none green (as current), nor red/yellow color

**Comment by omnisci3nce** [comment]
Would you be open to someone contributing this?
I'm new to Rust so looking for some small issues to work on to get my feet wet and I just found your tool and really like it.

**Comment by mrjackwills** [comment]
Yeah absolutely happily accept any PRs, don't worry about being a noob, I still am one

**Comment by omnisci3nce** [comment]
Fantastic. I'll have a crack at it this weekend.

**Comment by omnisci3nce** [comment]
I will be setting up a server + bunch of Docker stuff at work in the next few weeks so this TUI will come in handy. Until now I've just been using the commands as-is.

**Comment by omnisci3nce** [comment]
@mrjackwills Do you have a reliable way of getting into the Restarting state in the TUI? When I use 'restart' on my current docker containers as well as the three you have listed in the README, I do not see the state change to Restarting but rather I see

"running" .. "loading spinner starts" ......................... "see SIGTERM in logs" .. UI stays "running" but _maybe_ flashes a different state or text that I couldn't see

Or is implementing something which turns the UI into "Restarting..." until it goes down then comes back up part of the scope of this issue?

**Comment by omnisci3nce** [comment]
Thanks
